README.md

```diff
@@ -1,10 +1,36 @@
 # WeTransfer-iOS-CI
+[![Build Status](https://travis-ci.org/WeTransfer/WeTransfer-iOS-CI.svg?branch=master)](https://travis-ci.org/WeTransfer/WeTransfer-iOS-CI)
+
 Containing all the shared CI logic for WeTransfer repositories
 
-# Features
-- Integrate [SwiftLint](https://github.com/realm/SwiftLint) to lint source code and tests. We have a different config for tests to allow unwrapping for example
+# Why should I use it?
+What's in it for me? Well, quite a lot! With low effort to add it to your project.
+
+- Integrate [SwiftLint](https://github.com/realm/SwiftLint) to lint source code and tests. We have a different config for tests to allow `unwrapping optionals` for example
 - Integrate [Danger](http://danger.systems/) to automatically improve PR reviews
 
+## Danger features
+Following are a list of features which are posted in a comment on PRs based on the submitted files.
```

Comment:
Nit: This should be "Following is a list of features..." I believe. There is only one list.